### PR TITLE
Playground for the drawer component - docs

### PR DESCRIPTION
**Pull request by ddouglasz**
#### Summary

Playground for the drawer component required to write the drawer component docs.

### CODE REVIEW

File: website-components-playground/src/pages/drawer.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          name: 'title',
+          label: 'Title',
+          initialValue: 'Lorem ipsum',
+        },
+        {
+          kind: 'text',
+          name: 'title',
+          label: 'subTitle',
+          initialValue: 'Lorem ipsum dolor sit amet',
+        },
```
Comment (kark) [inline website-components-playground/src/pages/drawer.tsx:23]:
```suggestion
        {
          kind: 'text',
          name: 'subtitle',
          label: 'Subtitle',
          initialValue: 'Lorem ipsum dolor sit amet',
        },
```

**Review by kark — COMMENTED** [review]
Nice 🙌

File: website-components-playground/src/pages/drawer.tsx (hunk 1 above)
Comment (ddouglasz) [reply website-components-playground/src/pages/drawer.tsx]: updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3513/commits/6591cae5936d0d7665d36a5613ef3c3ac9abfde6
Thank you 🙌🏽

**Review by CarlosCortizasCT — APPROVED** [review]
Thanks Douglas 🙇

**Review by kark — APPROVED** [review]

**Pull request merged**